index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>My Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header class="navbar">
+      <a href="#">Dan Logo</a>
+      <span> <img src="images/burger.png" alt="toggle icon" /> </span>
+    </header>
+    <section class="header">
+      <div class="header-title">
+        <h1>Hey there. I'm Dan Elifeleti</h1>
+        <h2 class="green">I'm a software developer</h2>
+      </div>
+      <div class="header-content">
+        <p>
+          I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have a
+          project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="icon-style">
+        <ul class="icons">
+          <li>
+            <a href="#"><img src="images/git-logo.png" alt="cat icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/linkedin.png" alt="linkldn icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/vector.png" alt="two icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/twitter-logo.png" alt="twitter icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/m-logo.png" alt="m icon" /></a>
+          </li>
+        </ul>
+      </div>
+
+      <div class="anchor-arrow">
+        <img src="images/anchor-arrow.png" alt="arrow" />
+      </div>
+    </section>
+    <section class="project-container">
+      <div class="project-title">
+        <h2>My Recent Work </h2>
+      </div>
```

Comment:
Great job @Danelif. 👏

- [ ] [Optional] Please try your best to follow the Figma design.
- Your design
![works yours](https://user-images.githubusercontent.com/63917725/176770811-bb7aed22-0410-48fe-a259-a535c6132b24.PNG)

- Figma design
![works](https://user-images.githubusercontent.com/63917725/176770858-2d4889b1-2eea-4213-af4a-8733f1ab5d6f.PNG)